resource/schema/userdata.sql

```diff
@@ -205,6 +205,25 @@ CREATE TABLE collectionRelations (
 CREATE INDEX collectionRelations_predicateID ON collectionRelations(predicateID);
 CREATE INDEX collectionRelations_object ON collectionRelations(object);
 
+CREATE TABLE feeds (
+    libraryID INTEGER PRIMARY KEY,
+    name TEXT NOT NULL,
+    url TEXT NOT NULL UNIQUE,
+    lastUpdate TIMESTAMP,
+    lastCheck TIMESTAMP,
+    lastCheckError TEXT,
+    cleanupAfter INT,
+    refreshInterval INT,
+    FOREIGN KEY (libraryID) REFERENCES libraries(libraryID) ON DELETE CASCADE
+);
+
+CREATE TABLE feedItems (
+    itemID INTEGER PRIMARY KEY,
+    guid TEXT NOT NULL UNIQUE,
+    readTimestamp TIMESTAMP,
```

Comment:
`readTime`? (Also, we're not actually using the time, are we? Unless there's a compelling need to show or use that anywhere, I think this can just be `read INT` with a boolean.)